src/platform_impl/linux/event_loop.rs

```diff
@@ -351,7 +351,10 @@ impl<T: 'static> EventLoop<T> {
               window.input_shape_combine_region(None)
             };
           }
-          WindowRequest::WireUpEvents { transparent } => {
+          WindowRequest::WireUpEvents {
+            transparent,
+            cursor_moved,
+          } => {
             window.add_events(
               EventMask::POINTER_MOTION_MASK
                 | EventMask::BUTTON1_MOTION_MASK
```

Comment:
We could probably remove these Event masks instead and the event shouldn't be triggered at all

Reply:
Hmm I still got the event despite if I disable it

Reply:
Oh, nevermind I only disabled `BUTTON1_MOTION_MASK`. But there are other events like holding mouse button and then dragging can still trigger lots of events. I prefer we check it in connect signal callback for all.

Reply:
sure